python/test/unit/hopper/test_gemm.py

```diff
@@ -430,15 +443,17 @@ def grid(META):
         B_ORDER_0=b_order[0], B_ORDER_1=b_order[1],  #
         W_ORDER_0=w_order[0], W_ORDER_1=w_order[1],  #
         Z_ORDER_0=z_order[0], Z_ORDER_1=z_order[1],  #
-        num_warps=NUM_WARPS, num_ctas=NUM_CTAS, num_stages=NUM_STAGES)
+        num_warps=NUM_WARPS, num_stages=NUM_STAGES, **kwargs)
 
     torch.set_printoptions(profile="full")
     golden = torch.nn.functional.normalize(golden)
     z = torch.nn.functional.normalize(z)
     assert_close(z, golden, rtol=1e-2, atol=1e-3, check_dtype=False)
 
-    disable_mmav3 = os.environ.get('DISABLE_MMA_V3', 'not found').lower()
-    if disable_mmav3 not in ["on", "true", "1"] and BLOCK_M >= 64 and NUM_CTAS == 1 and BLOCK_N <= 256:
-        ptx = pgm.asm['ptx']
-        wgmma_n = int(max(BLOCK_N / max(NUM_WARPS / max(BLOCK_M / 16, 1), 1), 8))
-        assert re.search(r'wgmma.mma_async.sync.aligned.m\d+n{}k16(?:.row.col)?.f32.f16.f16'.format(wgmma_n), ptx)
+    # check is cuda backend specific
```

Comment:
You can do the following to early return on hip
if is_hip():
  return

Reply:
Thanks. changed.